sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 5.4.0 (Unreleased)
+## 5.4.0 (2021-02-09)
 
 - A helper method `parseEventHubConnectionString` has been added which validates and
```

Comment:
Nit: Can we move things around in the order of importance of how this update affects the user? I would go with
- custom end point address
- parse connection string
- RetryMode
- update docs

Reply:
Yup, moved things around!